### ISSUE TITLE
Баг с подгрузкой шрифтов LocalStorage в Google Chrome

### ISSUE THREAD

**Issue by kolyaraketa** [agragregra/start_html#16]
Заметил один очень неприятный баг: в хроме не хотят грузиться шрифты в локал-сторедж. В эдже и фаэрфоксе все ок, шрифты грузятся отлично. Но если шрифты подключаться как и стили, тоесть через `<script>loadCSS( "css/fonts.min.css?ver=1.0.0", false, "all" );</script>`, то шрифты подключаются и отрабатывают нормально. Очень странное поведение, не сразу понял, почему в хроме не хочет работать локал сторедж.

Не знаю, зачем создал ишью, но может кому-то будет очень полезно, может что-то нужно подправить в коде. Очень сранно что до меня никто не заметил этот баг.

**Comment by wdmcourses** [comment]
Сайт должен быть в корне домена. А не в папке.
После пертурбаций со шрифтами, надо чистить локалсторидж localStorage.clear()
Код рабочий.

**Comment by kolyaraketa** [comment]
Да, действиетельно, очистка локал-стореджа помогает. Очень часто приходится верстать какой-то лендинг, который на сайте не один, и он располагается в подпапке. Тогда не нужно ложить шрифты в локал-сторедж?

ПыСы. Стыдно что не додумался очистить локалсторедж.

**Comment by wdmcourses** [comment]
Я обычно для работы в папке комментирую строку и дублирую просто loadCSS, как вы описали в первом сообщении. На продакшене возвращаю обратно.

**Comment by kolyaraketa** [comment]
Спасибо большое за разъясения. Может быть у кого-то еще возникнут вопросы - будет где посмотреть.

**Comment by WhoAm7i7** [comment]
в одном из видео Алексей еще на этом акцентировал внимание, буквально на днях, пересматривал....
Ракета спасибо за поднятый вопрос, буду еще внимательней к этому аспекту.
Касательно чистки localStorage, не совсем понимаю, как технически это сделать, буду признателен за разъяснение.

**Comment by kolyaraketa** [comment]
Почистить можно в браузере в консоли, просто вводишь команду и все. Для меня это не вариант, поскольку я работаю в компании маркетинговой, и там на одном домене куча лендингов, все они разные и не похожие друг на друга, каждый верстается отдельно, шрифты все разные, и каждый лендос в отдельной папке как отдельный проект. Пожалуй откажусь от localStorage.
Насчет видео с информацией по шрифтам - может я был еще совсем зеленый и пропустил мимо ушей, а может и видос пропустил, но это наверное видео было давненько уже, в последних я такого не помню. 
А так шаблон оличный, я его под себя подстроил, добавил несколько плагинов которые всегда использую, и тягаю от проекта в проект. Спасибо огромное за разьяснение и удачи)

**Comment by WhoAm7i7** [comment]
согласен, шаблон отличный, автору отдельное за это спасибо!
у себя можно почистить, а у клиент как, на сколько я правильно понял логику localStorage основная ее задача быстрое отображение самой страницы на стороне клиента + отличная оценка PageSpeed
Если не прав поправь.

**Comment by wdmcourses** [comment]
WhoAm7i7, на продакшене багов быть не должно. У клиента будет результат вашей работы, который не должен содержать подобных ошибок.

**Comment by WhoAm7i7** [comment]
agragregra, только вхожу на уровень полноценной верстки, до этого баловался сборкой и правкой сайтов на WP, с первого клиента отработанного по твоей системе и урокам, перечислю 10% от полученных денег, мне будет приятно отблагодарить тебя... не останавливайся в развитии своего YouTube канала и наполнении его поистине качественным контентом в создании новых проектов по верстке и посадки на CMS.... для многих это важно....

**Comment by inamax335** [comment]
WhoAm7i7, отчистить localStorage можно командой localStorage.clear(); в консоли браузера.

![screenshot_7](https://cloud.githubusercontent.com/assets/20101859/17019505/d1ca9e22-4f66-11e6-9d02-2dd67aa534ac.png)

**Comment by WhoAm7i7** [comment]
inamax335 за наглядное отображение, спасибо!

**Comment by pashog** [comment]
А если уж накосячили, например на стейдже, или на своём тестовом сервере, который отдаёте на проверку клиенту, то можно его попросить вставить в адресную строку `javascript: localStorage.clear();` и нажать `Enter`. Для клиента это куда проще, чем лезть в консоль.

Я так вообще добавил себе закладку c такой урлой на панель закладок, чтобы нигде ничего не прописывать ;)

**Comment by wdmcourses** [comment]
Можно localStorage.clear(); разместить в common.js

**Comment by pashog** [comment]
> Можно localStorage.clear(); разместить в common.js

Ну это уж слишком) Главное, не забыть потереть перед релизом, если уж так)

**Comment by wdmcourses** [comment]
О том и речь.

**Comment by WhoAm7i7** [comment]
народ offtopic
подскажите как подключить форму ОСТАВИТЬ ЗАЯВКУ в разделе "Контакты" на сайте http://vkgroup.kiev.ua/ ? 
перепробовал разные комбинации, рациональней спросить, как правильно, чем описывать все процессы которые этому сопутствовали...
